knowledge-base/common-animations-decrease.md

```diff
@@ -1,83 +1,108 @@
 ---
-title: Remove animations of dropdowns or make them faster
+title: Remove DropDown Animations or Make Them Faster
 description: How to remove the dropdown animations or increase their speed
 type: how-to
-page_title: Remove animations of dropdowns or make them faster
+page_title: How To Remove DropDown Animations or Make Them Faster
 slug: common-kb-animations-decrease
 position: 
-tags: 
+tags: telerik, blazor, animations, css
+ticketid: 1637172
 res_type: kb
 ---
 
 ## Environment
+
 <table>
-	<tbody>
-		<tr>
-			<td>Product</td>
-			<td>Grid for Blazor</td>
-		</tr>
-	</tbody>
+    <tbody>
+        <tr>
+            <td>Product</td>
+            <td>UI for Blazor</td>
+        </tr>
+    </tbody>
 </table>
 
 
 ## Description
-I want to make the animations of the Telerik dropdowns faster for fast data entry.
 
-I don't want animations for the dropdowns to accommodate users who prefer no animations (e.g., for accessibility reasons).
+This knowledge base article discusses the following scenarios:
 
-## Solution
-You can use a CSS rule override to change the animation the Telerik dropdowns use.
+* How to make the animations of the Telerik dropdowns faster for fast data entry?
+* How to remove all animations for Telerik Blazor ComboBox, DropDownList, DateTimePicker and other selection components?
+* I don't want animations for the dropdowns to accommodate users who prefer no animations (e.g., for accessibility reasons).
```

Comment:
Consider re-writing the statement for clarity. Right now, it could easily be interpreted as "I want to keep my animations and ignore users who don't want them.". Consider something like:
```suggestion
* I want to remove animations for the dropdowns to accommodate users who prefer no animations (e.g., for accessibility reasons).
```